cli/bpmetadata/types.go

```diff
@@ -68,11 +75,54 @@ type BlueprintDescription struct {
 	Architecture []string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
 }
 
+type BlueprintDeploymentTime struct {
+	Configuration int `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+	Deployment    int `json:"deployment,omitempty" yaml:"deployment,omitempty"`
+}
+
+type BlueprintCostEstimate struct {
+	Description string `json:",omitempty" yaml:",omitempty"`
+	Url         string `json:",omitempty" yaml:",omitempty"`
+}
+
+type BlueprintCloudProduct struct {
+	ProductId   string
+	PageUrl     string
+	Label       string
+	LocationKey bool
+}
+
+type BlueprintQuotaDetail struct {
```

Comment:
Could you add some comments for these quota fields as I am not sure I understand what each may contain.

Reply:
Added. There's more context in internal docs as well:
- https://docs.google.com/document/d/1BGuladCll3jHi6PbPc4_W7SYd1hgHf5jS7lTnBMvAxY/edit#bookmark=id.p8qj3jb77xxm 
- https://docs.google.com/document/d/1esH1b1m-x1ue3hxmHvdXRP5X3Nl4YUkVCfk-9pDZ2ww/edit?resourcekey=0-A8rjUT057NzU7pPLVI9kFw

Reply:
Updated the type for quotas based on our discussion.

Reply:
Thanks LGTM!